build/docker-extension/docker-compose.yml

```diff
@@ -0,0 +1,20 @@
+version: '3'
+
+services:
+  portainer:
+    image: ${DESKTOP_PLUGIN_IMAGE}
+    command: ["--admin-password", "$$$$2a$$$$10$$$$HX4qSZhtQcvKUNmAsPXuPe9POkM7gdaFPcSnRjokgb8EkMI.1gkSa"]
+    # command: ["--generate-password"]
```

Comment:
this is not used? so we can remove `flags.GenerateAdminPassword` and what was added for that?

Reply:
IIRC we will be using the generate flag instead of supplying a hardcoded pass like this, so that passwords will be unique for each user

Reply:
well, the docker-compose says something else

Reply:
we are keeping the default admin and password for extension and scope the network to localhost only

Reply:
so that means that `--generate-password` is useless, right?

Reply:
yes based on discussion yesterday, for v1 we will not autogen a password and can look at this again later if we need it

Reply:
resolving as autogen code removed